src/main/java/tfifteenfour/clipboard/ui/StudentCard.java

```diff
@@ -46,8 +52,13 @@ public StudentCard(Student student, int displayedIndex) {
         this.student = student;
         id.setText(displayedIndex + ". ");
         name.setText(student.getName().fullName);
-        studentId.setText(student.getStudentId().value);
-
+        //studentId.setText(student.getStudentId().value);
```

Comment:
LGMT! 

Probably nothing to hold for this statement already so it is good to remove.